examples/examples-cpu/nyc-taxi-snowflake/dashboard.ipynb

```diff
@@ -61,13 +61,12 @@
     "import datetime as dt\n",
     "import numpy as np\n",
     "\n",
-    "import hvplot.dask, hvplot.pandas\n",
```

Comment:
These imports are needed they extend the dask.Dataframe and pandas.DataFrame classes to support `.hvplot`.

Reply:
oh ok. I guess that would explain the `#noqa` in these examples :grinning: --> https://hvplot.holoviz.org/user_guide/Plotting.html#plotting

Reply:
put them back: https://github.com/saturncloud/examples/pull/60/commits/4c721a4d24a58202c9c751622e6c95bc75a50d1a